src/compass/utils/validate_product.py

```diff
@@ -158,15 +157,15 @@ def _compare_static_layer_rasters(file_ref, file_sec, static_layer_items):
     static_layer_items: list[str]
         List of names of static layers to compare
     """
-    grid_path = f'{DATA_ROOT}/CSLC/grids/static_layers'
+    data_path = f'{DATA_PATH}/static_layers'
     with h5py.File(file_ref, 'r') as h_ref, h5py.File(file_sec, 'r') as h_sec:
         for static_layer_item in static_layer_items:
             if static_layer_item == 'layover_shadow_mask':
                 continue
 
             # Retrieve static layer raster from ref and sec HDF5
-            slc_ref = h_ref[f'{grid_path}/{static_layer_item}']
-            slc_sec = h_sec[f'{grid_path}/{static_layer_item}']
+            slc_ref = h_ref[f'{data_path}/{static_layer_item}']
+            slc_sec = h_sec[f'{data_path}/{static_layer_item}']
```

Comment:
```suggestion
            static_path = f'{data_path}/{static_layer_item}'
            slc_ref = h_ref[static_path]
            slc_sec = h_sec[static_path]
```
Declare the path just once. Sorry missed this in the last go through.